zip-0302.rst

```diff
@@ -45,13 +45,27 @@ contents of a memo. It does not define consensus requirements.
      + strip any trailing zero bytes
      + decode it as a UTF-8 string (if decoding fails then report an error).
 
++ If the first byte has a value of 0xF5, then the reader MUST:
+     + Interpret the next few bytes (1 to 9 of them) as a 64-bit unsigned variable-length
+       integer [#Bitcoin-CompactSize]_, and use it as an arbitrary application-defined
+       "type" field.
+     + Interpret the next 1 or 2 bytes as a CompactSize value constrained to the range 0..510, and use
+       the resulting value as the length of the remaining data.
+     + If 1 + the number bytes used for the type field + the number of bytes used for the
+       length field + the length > 512 then report an error.
+     + Inspect the padding after the end of the indicated length, and if it
+       contains anything other than bytes of value 0x00 then report an error.
```

Comment:
This was present in the original ZIP 302 PR, but I now notice that it may not actually match what has been implemented in the ecosystem. For example, my implementation of this from 2019 (https://github.com/zcash/librustzcash/pull/668) allows for sequential TLV tuples.